OddNumberSum.c

```diff
@@ -0,0 +1,40 @@
+#include <stdio.h>
+
+int main()
+
+{
+   int list[] = { 7, 4, 1, 5, 3, 2, 7, 6, 4, 2, 9, 1, 2, 6, 8, 2, 3, 4 };
+
+   int array = sizeof(list);
+
+   int Size = sizeof(list[0]);
+
+   int lenghtOfArray = array / Size;
+
+   int evenNumber = 0;
+
+   int oddNumber = 0;
+
+   for(int index = 0; index < lenghtOfArray; index++)
+   {
+      if((index % 2) == 0)
+      {
+         evenNumber += list[index];
+      }
+
+      else if((index % 2) != 0)
```

Comment:
```suggestion
      else
```

You can leave out the `if` part because all integers that aren't even are odd.